Sources/App/Views/Utils.swift

```diff
@@ -0,0 +1,12 @@
+
+func pluralize(count: Int, singular: String, plural: String? = nil) -> String {
```

Comment:
Missing a test for this?

Reply:
Yes, I rushed to push when I saw you're active :)

Reply:
I'll add some real quick